daemon/cmd/daemon_main.go

```diff
@@ -735,6 +735,9 @@ func initializeFlags() {
 	flags.String(option.ProxySocketDir, defaults.ProxySocketPath, "Directory path where proxy sockets are located")
 	option.BindEnv(Vp, option.ProxySocketDir)
 
+	flags.Bool(option.EmbeddedProxyEnabled, true, "Whether the embedded Cilium Proxy should be enabled or not")
```

Comment:
what do you think about external-envoy-proxy ?

Reply:
@sayboras changed to `external-envoy-proxy`